runners/gradle-plugin/build.gradle.kts

```diff
@@ -118,3 +126,7 @@ afterEvaluate { // Workaround for an interesting design choice https://github.co
     configureSonatypePublicationIfNecessary("pluginMaven", "dokkaGradlePluginPluginMarkerMaven")
     createDokkaPublishTaskIfNecessary()
 }
+
+tasks.processResources {
+    duplicatesStrategy = DuplicatesStrategy.WARN
+}
```

Comment:
I see that it migrated from another place. Any idea what this was/is used for? Are there any specific problems if it gets deleted?

Reply:
If you run `./gradlew assemble` you can see the warnings

I don't know about this one:

```
> Task :runners:maven-plugin:jar
Encountered duplicate path "META-INF/maven-plugin.kotlin_module" during copy operation configured with DuplicatesStrategy.WARN
```



This warning should probably be resolved by deleting `META-INF/gradle-plugins/org.jetbrains.dokka.properties` - the plugin identifier is provided by the Gradle config, 

```
> Task :runners:gradle-plugin:processResources
Encountered duplicate path "META-INF/gradle-plugins/org.jetbrains.dokka.properties" during copy operation configured with DuplicatesStrategy.WARN
```